packages/manager/apps/web/client/app/private-database/dashboard/private-database.routing.js

```diff
@@ -0,0 +1,98 @@
+import controller from './private-database.controller';
+import template from './private-database.html';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.private-database.dashboard', {
+    url: '/:productId',
+    template,
+    controller,
+    controllerAs: 'ctrl',
```

Comment:
maybe $ctrl if its not too much of refactoring work.

Reply:
I think this could be done but should be in a separate PR as this one is already updating quite a lot

Reply:
See my previous comment :)